cmake/script/GenerateHeaderFromJson.cmake

```diff
@@ -5,10 +5,10 @@
 file(READ ${JSON_SOURCE_PATH} hex_content HEX)
 string(REGEX MATCHALL "([A-Za-z0-9][A-Za-z0-9])" bytes "${hex_content}")
 
-file(WRITE ${HEADER_PATH} "#include <string>\n")
+file(WRITE ${HEADER_PATH} "#include <string_view>\n")
 file(APPEND ${HEADER_PATH} "namespace json_tests{\n")
 get_filename_component(json_source_basename ${JSON_SOURCE_PATH} NAME_WE)
-file(APPEND ${HEADER_PATH} "static const std::string ${json_source_basename}{\n")
+file(APPEND ${HEADER_PATH} "inline constexpr char detail_${json_source_basename}_bytes[]{\n")
```

Comment:
What does the `detail_` prefix mean?

Reply:
Ok, I hadn't come across this yet. I don't have a preference, you can leave it as is.